### PR TITLE
jo: add recipe for 1.6

### PR DESCRIPTION
**Pull request by augiedoggie**

### CODE REVIEW

File: app-text/jo/jo-1.6.recipe [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	cmd:autoreconf
+	cmd:awk
+	cmd:find
+	cmd:gcc
+	cmd:grep
+	cmd:install
+	cmd:make
+	cmd:pkg_config
+	cmd:sed
+	"
```
Comment (Begasus) [inline app-text/jo/jo-1.6.recipe:39]: Are all these needed? pkg-config for instance is only needed to find pandoc (which we don't have): https://github.com/jpmens/jo/blob/master/configure.ac#L44

Comment (augiedoggie) [reply app-text/jo/jo-1.6.recipe:39]: pkg-config can be removed but I believe the rest are needed.

Comment (augiedoggie) [reply app-text/jo/jo-1.6.recipe:39]: Actually, the build system requires pkg-config or it fails.  I can remove autoheader though.

Comment (Begasus) [reply app-text/jo/jo-1.6.recipe]: Haven't checked it (yet), thanks for looking into it :)

**Pull request merged**